CHANGELOG.md

```diff
@@ -48,6 +48,13 @@ _None._
 
 _None._
 
+## 2.18.1
+
+### Bug Fixes
+
+- `annotate_test_failures` no longer prints an empty `<code>` box when reporting a `failure` with no extra `details`. [#63]
+- Fix an issue with `annotate_test_failures` on CI steps that are using the `parallelism` attribute. [#67]
+
```

Comment:
Note: I've decided to directly add the CHANGELOG entry under `2.18.1` (and not under "Unreleased") because I plan to do a new release and tag of the plugin immediately after this PR lands.